MoonDawgArduino/Libraries/Movement/Movement.cpp

```diff
@@ -0,0 +1,126 @@
+// Cameron Schwartzberg
+// 2/18/2024, 7:37 PM
+// 240216-212839-uno
+// (Description)
+// (Contributors)
+
+#include "Movement.h"
+#include <Arduino.h>
+
+constexpr long rampTime = 1000;
+uint8_t speed = 90;
+// Add ramping functions, so we don't have to pass speed
+// Dig motor movement, will write to the motor, "180" for forward, "0" for backward, "90" for stop
+// Actuator movement, will ...

```

Comment:
actuator movement will?

Reply:
This is old and will be updated to reflect how they work